consensus/validator_test.go

```diff
@@ -118,7 +119,30 @@ func TestValidateBlock(t *testing.T) {
 				s, r, err := c.verifyBlock(blk, state, 0)
 				assert.Nil(t, s)
 				assert.Nil(t, r)
-				assert.Equal(t, fork.ErrMaxFeePerGasTooLow, err)
+				expectedErr := errors.New("max fee per gas is less than block base fee")
```

Comment:
Also here, is there any reason why we want the error msg in the fork package to be private and not public anymore? We could expect the error from the fork package here and on line 145

Reply:
As far as I see it is only used by its file and by this tests.

IMO the `var` should be package-scoped since there is no actual feature outside of the `fork` package using it.

If you feel strongly about keeping it public I do not mind changing it back, however the new error will follow the same criteria (so public too).